Lib/test/test_docxmlrpc.py

```diff
@@ -192,6 +193,20 @@ def test_annotations(self):
              b'method_annotation</strong></a>(x: bytes)</dt></dl>'),
             response.read())
 
+    def test_server_title_escape(self):
+        self.serv.set_server_title('test_title<script>')
+        self.serv.set_server_documentation('test_documentation<script>')
+        self.assertEqual('test_title<script>', self.serv.server_title)
+        self.assertEqual('test_documentation<script>',
+                self.serv.server_documentation)
+
+        # bpo-38243
```

Comment:
Hum, please elaborate the comment. Something like:

"# bpo-38243: Ensure that the server title and documentation are escaped for HTML."

Would you mind to move the comment at the start of the function?